### DISCUSSION TITLE
Bad underline cursor visibility

### DISCUSSION THREAD

**Discussion by hikamaree** [ghostty-org/ghostty#3741] [Feature Requests, Ideas]
When I edit an underlined word, the cursor is not visible at all because it looks the same as the underline.
I tried to fix this by changing the value of adjust-underline-position but again the underline and cursor were rendered at the same height.
Here is the difference between alacritty and ghostty.
![Dec28_124543](https://github.com/user-attachments/assets/1308df5b-55bf-4bb5-b5b4-5b8f55634982)
![Dec28_124721](https://github.com/user-attachments/assets/516d8677-de92-4304-9fd5-068548d4085f)

**Comment by zx8** [comment]
Related/duplicate? https://github.com/ghostty-org/ghostty/discussions/3194

**Reply by hikamaree** [reply]
It looks like underline cursor thickness has the same value as ```adjust-underline-thickness```. Underline cursor also follows ```adjust-underline-position``` value and that make it look same as underline.
